### DISCUSSION TITLE
Material UI with static export

### DISCUSSION THREAD

**Discussion by ektarathi** [vercel/next.js#24419] [Help]
Hi,

When I am trying to do the next export for static site generation using Material UI, the Material UI styles are not loading on index.html.

Any pointers will be appreciated?

Thanks ,
Ekta

**Comment by apollonian** [comment]
Could you add a reproducible example or a CodeSandbox? That would be very helpful, thanks!

**Reply by ektarathi** [reply]
Hi,

I am running the code locally. It's not yet deployed to any server so not sure what exactly shall I send over to you so that you can have a look ?

I can attach over the exported folder here if that's helpful for you .

Thanks,

**Reply by apollonian** [reply]
Sure, that works!

**Reply by ektarathi** [reply]
Hi,

Please find attached screenshot and the zip folder with the code base.

<img width="1918" alt="Screenshot 2021-04-25 at 10 57 58" src="https://user-images.githubusercontent.com/11335129/115989242-5e054780-a5b5-11eb-8b15-2abfcddb161c.png">
[_static.zip](https://github.com/vercel/next.js/files/6371697/_static.zip)

**Reply by apollonian** [reply]
Seems to be working fine for me! You might need to check if you're serving all the assets from the out directory.

<img width="1603" alt="Screenshot 2021-04-25 at 4 56 18 PM" src="https://user-images.githubusercontent.com/2150306/115991657-2b744280-a5e7-11eb-82d8-61ca60fdefed.png">

**Reply by ektarathi** [reply]
I haven't used any assets so far...just implemented the Material UI styling to the containers block.

What about the error in the console? Are you also getting the same ?

**Reply by apollonian** [reply]
> Are you also getting the same ?

Nope!. I just did a `npx serve _static`.
